### DISCUSSION TITLE
Improvement to COM References list

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#592] [General]
<img width="743" alt="tbReferenceImprovements1" src="https://user-images.githubusercontent.com/25738109/147062056-d0e1001a-a805-4b72-b269-6718d7a78905.png">

The COM references list will now display more detailed information upon clicking an element, from v0.13.54 onwards.

**Comment by mburns08109** [comment]
VBA Gripe: I sure hope those windows are all RESIZABLE! having any of those paths/etc. get cut off or self-reducing with ellipses is NOT acceptable!
I also hope that text is selectable for copy/paste?

**Reply by DavidHollman** [reply]
There actually is a cool hack at least in the VB6 IDE to modify the dialog sizes: https://stackoverflow.com/a/54079227/3195477

**Reply by WaynePhillipsEA** [reply]
Yup, don't worry, the paths split onto multiple lines when necessary, and yes copy will work

**Comment by WaynePhillipsEA** [comment]
Further improvements in v0.13.55...

Search bar is now more discoverable:
<img width="370" alt="tbReferenceImprovements2" src="https://user-images.githubusercontent.com/25738109/147209708-6cacdba8-36e0-4350-9f7a-57d90166ffc7.png">

Library symbol / namespace is shown in the detailed information:
<img width="528" alt="tbReferenceImprovements3" src="https://user-images.githubusercontent.com/25738109/147209759-780f6b8b-b2d5-4595-8168-1ece6169b677.png">
